tensorflow/lite/micro/python/interpreter/tests/interpreter_test.py

```diff
@@ -73,6 +73,10 @@ def testCompareWithTFLite(self):
       # TODO: Remove tolerance when the bug is fixed.
       self.assertAllLessEqual((tflite_output - tflm_output), 1)
 
+      # Reset the model state
+      tflm_interpreter.reset()
+      tflite_interpreter.reset_all_variables()
```

Comment:
I'd say don't add reset here because the conv model is stateless so resetting here doesn't do anything and will just cause confusion. 

`tflm_interpreter.reset()` and `tflite_interpreter.reset_all_variables()` are also not equivalent.

Reply:
Got it. Should we have `TfLiteStatus ResetVariableTensors()` included as well? But it seems that we are going to remove it in the future: https://github.com/tensorflow/tflite-micro/blob/639ae6378a5847a610e32926459c632c0b68fc17/tensorflow/lite/micro/micro_interpreter.h#L125